website/pages/blog/introducing-the-tailscale-source-plugin.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Introducing The Tailscale Source Plugin
+tag: announcement
+date: 2022/12/14
+description: >-
+  CloudQuery is an open-source, extensible framework that gives you a
+  single-pane-of-glass to your cloud-infrastructure using SQL. Today, we are
+  happy to announce the release of the Tailscale source plugin.
+author: alex
+--- 
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+CloudQuery is an open-source, extendable framework that gives you a single-pane-of-glass to your cloud-infrastructure using SQL.
+Today, we are happy to announce the release of the Tailscale source plugin for CloudQuery.
+
+[Tailscale](https://tailscale.com/) is a company that provides open-source software defined mesh virtual private network (VPN) software and a web-based management service.
```

Comment:
Similar to https://github.com/cloudquery/cloudquery/pull/5643#discussion_r1048624571

Maybe we can write:
```
If you're not familiar with the Tailscale, we recommend you visit their [homepage](https://tailscale.com/
```

So we don't have to worry about the description getting stale

Reply:
I suppose at the very least a short intro is a must.
I'll make it shprter though.